### PR TITLE
[ICDS Dashboard] Made domain name static

### PR DESCRIPTION
**Pull request by challabeehyv**
Made domain name static in the dashboard usage report

##### SUMMARY
Made domain name static in the dashboard usage report

##### FEATURE FLAG
No

### CODE REVIEW

File: custom/icds_reports/sqldata/exports/dashboard_usage.py [hunk 1]
```diff
@@ -189,7 +189,7 @@ def get_excel_data(self):
 
             usernames = [user['username'] for user in users if dashboard_uname_rx.match(user['username'])]
 
-            records = list(ICDSAuditEntryRecord.objects.filter(url='/a/icds-dashboard-qa/cas_export',
+            records = list(ICDSAuditEntryRecord.objects.filter(url='/a/' + self.domain + '/cas_export',
```
Comment (Rohit25negi) [inline custom/icds_reports/sqldata/exports/dashboard_usage.py]: this looks good. just two comments.
1) using format could be more readable
2) can you change `cas_export` to `icds_export_indicator`. the Url was changed after you started building the report.
@challabeehyv

**Review by Rohit25negi — CHANGES_REQUESTED** [review]
Left 1 small comment.

File: custom/icds_reports/sqldata/exports/dashboard_usage.py (hunk 1 above)
Comment (challabeehyv) [reply custom/icds_reports/sqldata/exports/dashboard_usage.py]: @Rohit25negi  should i harcode the 'icds_export_indicator' value or is there a way i can get it in dynamic?

Comment (Rohit25negi) [reply custom/icds_reports/sqldata/exports/dashboard_usage.py]: you can hardcode its a url

Comment (challabeehyv) [reply custom/icds_reports/sqldata/exports/dashboard_usage.py]: done.please check

**Review by Rohit25negi — APPROVED** [review]

**Pull request merged**